content/blog/2017-11-28-fragments-in-react-v16.2.md

```diff
@@ -0,0 +1,203 @@
+---
+title: "Fragments in React v16.2"
+author: [clemmy]
+---
+
+Today, we're excited to improved support for fragments in React! We are introducing a new `Fragment` export, which allows
+which allows users to render *static* fragments using familiar JSX element syntax.
+
+```jsx
+import React, { Fragment } from 'react';
```

Comment:
Would this work for TS users? AFAIK they have to enable a special mode to get imports like this working. This is a thorny issue: https://github.com/facebook/react/issues/11503. That's why I previously suggested to remove all imports from this blog post and just use `React.Fragment`. People will import React they way they already do, and there's no need to confuse them with a syntax that might not work.

Reply:
It also doesn't match the Flow recommendation. Note they change default React import to a namespace import as part of conversion: https://flow.org/en/docs/react/components/. I just don't think we should confuse people with import syntax here. The blog post is to about it.